README.md

```diff
@@ -1,5 +1,5 @@
 <h1 align="center">
-  <a href="https://toggl.com"><img src="https://toggl.com/site/images/media-toolkit/logo_02-644bd26148b73c19d9c91e5baecd8e31.jpg" alt="Toggl" width="200"></a>
+  <a href="https://toggl.com"><img src="https://raw.githubusercontent.com/toggl-open-source/toggldesktop/gh-pages/assets/toggl-track-128x128.png" alt="Toggl Track" width="200"></a>
```

Comment:
let's fix the width so that it matches the actual image size
```suggestion
  <a href="https://toggl.com"><img src="https://raw.githubusercontent.com/toggl-open-source/toggldesktop/gh-pages/assets/toggl-track-128x128.png" alt="Toggl Track"></a>
```